torrt/toolbox.py

```diff
@@ -301,10 +302,10 @@ def walk(forced=False, silent=False, remove_outdated=True):
         )
 
 
-def update_torrents(hashes, remove_outdated=True):
+def update_torrents(to_check, remove_outdated=True):
```

Comment:
Such a change needs to be done in a backward compatible manner.
Someone might still use this function directly passing a list.